packages/zoe/src/contractFacet/zcfSeat.js

```diff
@@ -335,7 +337,7 @@ export const createSeatManager = (
         );
         return amountKeywordRecord;
       },
-      clear: ignoreContext(clear),
+      clear: ({ self }) => clear(self),
```

Comment:
just parroting the linter here: gotta remove the `ignoreContext` import

Reply:
though, why delegate here at all?
```suggestion
      clear: ({ self }) => {
          if (zcfSeatToStagedAllocations.has(self)) {
            zcfSeatToStagedAllocations.delete(self);
          }
      };
```

Reply:
I think MarkM hinted at the right choice here. Please inline the method, so it'll be more apparent that the parameter handling is parallel to the sibling methods.